src/pickups/components/CurrentUser.vue

```diff
@@ -34,11 +36,19 @@ export default {
       type: Object,
       required: true,
     },
+    pickup: {
+      type: Object,
+      required: true,
+    },
   },
   components: {
     ProfilePicture,
   },
-
+  methods: {
+    inProgress (pickup) {
```

Comment:
if you need this in other places as well, makes sense to move it into the `enrich` getter

could be renamed to `ongoing` to unify with locale message